cmake/README.md

```diff
@@ -10,6 +10,8 @@ TBB_SANITIZE:STRING - Sanitizer parameter, passed to compiler/linker
 TBB_SIGNTOOL:FILEPATH - Tool for digital signing, used in post install step for libraries if provided.
 TBB_SIGNTOOL_ARGS:STRING - Additional arguments for TBB_SIGNTOOL, used if TBB_SIGNTOOL is set.
 TBB4PY_BUILD:BOOL - Enable  Intel(R) oneAPI Threading Building Blocks (oneTBB) Python module build (OFF by default)
```

Comment:
Have you considered dependencies? E.g. TBB4PY depends on TBB: https://github.com/oneapi-src/oneTBB/blob/9e15720bc7744f85dff611d34d65e9099e077da4/python/CMakeLists.txt#L41

Reply:
If we try to configure oneTBB like this `cmake -DTBB4PY_BUILD=ON -DTBB_BUILD=OFF`, configuration will fail with error `non-existent target "tbb"`. I think that is OK to fail here because such command does not have sense and we let user to know about it.